version.go

```diff
@@ -0,0 +1,4 @@
+package main
+
+// Version is the current version of Glide
+const Version = "0.13.0-dev"
```

Comment:
I'd prefer to keep the visibility the same as it was previously as part of this change.

However since this is a main package, other's can't import it anyway.